app/models/enterprise.rb

```diff
@@ -259,9 +268,9 @@ def relatives
     ", id, id)
   end
 
-  def plus_relatives_and_oc_producers(order_cycles)
+  def plus_parents_and_order_cycle_producers(order_cycles)
     oc_producer_ids = Exchange.in_order_cycle(order_cycles).incoming.pluck :sender_id
-    Enterprise.not_hidden.is_primary_producer.relatives_of_one_union_others(id, oc_producer_ids | [id])
+    Enterprise.not_hidden.is_primary_producer.parents_of_one_union_others(id, oc_producer_ids | [id])
```

Comment:
The old scope is now unused and can be removed.

Reply:
Hi, I have removed the unused scope and pushed the commit!

Reply:
Oh, I think that we were working on it at the same time and I did the same plus some spec fixes. Sorry to override your changes. It should be all good now though.